src/App.js

```diff
@@ -1,31 +1,29 @@
-import BlogDetails from './BlogDetails.js';
 import Components from './Components.js';
-import Features from './Features.js';
-import Home from './Home.js';
-import Navbar from './Navbar.js';
-import Sidebar from './Sidebar.js';
+import Navbar from './components/Navbar.js';
 import Toggle from './Toggle.js';
+import { Routes, Route } from 'react-router';
+import Home from './components/Home.js';
+import Create from './components/create.js';
+import NotFound from './components/NotFound.js';
+import About from './components/About-us.js';
+import BlogDetails from './components/BlogDetails.js';
+
 
 function App() {
   return (
     <div className="App">
+     <Navbar />
+
+     <Routes>
+      <Route path='/' element={<Home />} />
+      <Route path='/create' element={<Create />} />
+      <Route path='/about' element={<About />} />
+      <Route path='/components/:id' element={<BlogDetails />} />
+      <Route path='*' element={<NotFound />} />
 
-      <div className="content">
-        <h1 style={{
-          color: "rgb(218, 0, 181)",
-          fontWeight: "bolder",
-          fontFamily: "arial",
-          textAlign: "center",
-          marginTop: "1%",
-          textShadow: "0px 2px 2px black",
-        }}>My App Component</h1>
-        </div>
-      <Home/>
-      <Navbar />
-      <BlogDetails />
-      <Sidebar/>
+     </Routes>
+      
       <Toggle />
-      <Features/>
       <Components />
```

Comment:
These three components are outside the Routes tag which means they will always display in the DOM no matter the route you are in. I believe that is not what you want.

For example, you want the Components component to be loaded when the user clicks home or when the app starts initially. This is why you should remove it from here and nest it inside the Home component itself.

Reply:
oh...yes

Reply:
oh...yes, thank you sir. I wanted to ask before
I've tried to get rid of it but couldn't